Cargo.toml

```diff
@@ -29,3 +29,7 @@ members = [
 ]
 
 exclude = ["python"]
+
+[profile.release]
```

Comment:
Would these settings end up applying to any crate that uses `datafusion` as a dependency? Or only when a project is built within the `datafusion` workspace?